ragtime.jdbc/test/ragtime/jdbc_test.clj

```diff
@@ -6,11 +6,16 @@
             [clojure.java.io :as io]
             [clojure.java.jdbc :as sql]))
 
-(defn new-connection []
-  (sql/get-connection {:connection-uri "jdbc:h2:mem:"}))
+; By default, closing the last connection to an in-memory H2 database closes the database deleting all the content.
```

Comment:
I think we can remove this comment. If people are curious, the commit links back to the PR anyway, or they can look up the notation themselves.